dsa-23au/SoftwareSetup.md

```diff
@@ -106,7 +106,10 @@ Next to the "GitHub" integration, click on the three dots and choose "Manager th
 
 ### 7. Set your Git Config Email
 
-Run the `git-config.sh` script you set in the `GitHub` section above.
+Go to your `gitpod.io` environment variable settings at https://gitpod.io/user/variables.
+On this page add two new variables with the keys being
```

Comment:
Thanks @WinstonShine can you add two screenshots for these two steps showing what the user variables screen looks like, and examples of entering in the two variables?
Otherwise it looks good.